overlord/ifacestate/hotplug_test.go

```diff
@@ -238,15 +240,15 @@ func (s *hotplugSuite) TestEnsureUniqueName(c *C) {
 
 	names := []struct{ proposedName, resultingName string }{
 		{"foo", "foo"},
-		{"slot", "slot2"},
-		{"slot1", "slot2"},
-		{"slot1234", "slot1235"},
-		{"slot-1", "slot-3"},
-		{"slot3-5", "slot3-7"},
+		{"slot", "slot-3"},
```

Comment:
it feels like this tests could do with some more tweaking given that realistic input is more like "ifacexyz" or "devname" than "slot123"

Reply:
Done.